src/Android/Accessibility/AccessibilityHelpers.cs

```diff
@@ -161,18 +161,18 @@ private static string ExtractUri(string uri, AccessibilityNodeInfo addressNode,
             uri = browser.GetUriFunction(addressNode.Text)?.Trim();
             if (uri != null && uri.Contains("."))
             {
-                if (Uri.TryCreate(uri, UriKind.Absolute, out var uri2))
```

Comment:
Why not just call `return CoreHelpers.GetUri(uri)?.ToString() ?? uri;`

Reply:
Since the `Uri` class is a lot smarter than I am, I'm paranoid about the result of the string-to-uri-to-string not being identical to the original string, which is what we want to return here.